09-mapping.Rmd

```diff
@@ -122,6 +136,7 @@ map_nz1 = map_nz +
 ```
 
 Building on the previously created `map_nz` object the preceding code creates a new map object `map_nz1` which contains another shape (`nz_height`), representing high points (see Figure \@ref(fig:tmlayers), left).
+<!-- tm_bubbles explanation is missing -->
```

Comment:
tm_bubbles explanation is missing

Reply:
Well-spotted. Sorted now, even though `tm_bubbles()` is no longer used in the good. Definite good to mention though so thanks for pointing this out.